packages/design-system-docs/src/pages/components/Table/_Table.docs.scss

```diff
@@ -20,6 +20,79 @@ Markup: table-complex.example.html
 Style guide: components.table.complex
 */
 
+/*
+`<Table>`
+
+`This is a beta version of the Table components. The functionality and usage is subject to change in response to the feedback from design system users.`
+A `Table` component renders data table using semantic markup and ARIA. The table components allows for customization as needed by the users.
+
+@react-example Table.example.jsx
+
+@react-props Table.jsx
+
+Style guide: components.table.react
+*/
+
+/*
+`<TableBody>`
+
+A `TableBody` react component.
```

Comment:
I don't think these default descriptions are necessary

Reply:
Maybe instead of the template "A [react component name] react component", we could more specific description about what HTML element it corresponds to and how it should be used? Either way I imagine this should be brief.